docker/app.setup.sh

```diff
@@ -8,6 +8,9 @@ fi
 # Run WordPress docker entrypoint.
 . docker-entrypoint.sh 'apache2'
 
+echo "Delete the wp-config.php"
```

Comment:
What problem was this solving? Is this wp-config.php from a local WP that's already been running? So you want to remove it each time the app starts?

Reply:
When @kidunot89 and I were walking through things it seems like the php8 docker image already had a wp-config.php but we want to use ours that we generate with wp-cli, so this deletes the one from Docker and uses wp-cli to generate one with our creds.

Reply:
Was it this failure? https://github.com/wp-graphql/wp-graphql/runs/2975991532?check_suite_focus=true#step:8:40

Reply:
Ah yes. I think it was actually WP 5.7.2, not PHP 8. long day.

Reply:
There's a note in the WP docker hub page

> NOTE: Since March 2021, WordPress images use a customized wp-config.php that pulls the values directly from the environment variables defined above (see wp-config-docker.php in docker-library/wordpress#572 and docker-library/wordpress#577). As a result of reading environment variables directly, the cli container also needs the same set of environment variables to properly evaluate wp-config.php.

Looking at these now in case there's something else we'll need to consider.

Reply:
ok, so we can change our environment variable names to match and get rid of the delete/regenerate wp-config?